metron-platform/metron-enrichment/src/main/java/org/apache/metron/enrichment/bolt/EnrichmentJoinBolt.java

```diff
@@ -84,7 +88,7 @@ public JSONObject joinMessages(Map<String, JSONObject> streamMessageMap) {
     if(sourceType != null) {
       SensorEnrichmentConfig config = configurations.getSensorEnrichmentConfig(sourceType);
       if (config != null) {
-        return config.getEnrichmentFieldMap();
+        return config.getEnrichment().getFieldMap();
```

Comment:
I think that the enrichment variable is initialized, but a null check is never a bad idea.